src/mapeo-manager.js

```diff
@@ -214,4 +214,40 @@ export class MapeoManager {
     // TODO: Is it safe to do Promise.all here?
     return Promise.all(resultPromises)
   }
+
+  /**
+   * @param {string} projectId
+   * @returns {Promise<MapeoProject>}
+   */
+  async addProject(projectId) {
+    const existing = this.#activeProjects.get(projectId)
+
+    if (existing) return existing
```

Comment:
wondering about this return type a bit. should it return the project instance in general or some boolean about success, or void?

secondly, depending on the answer to this first question, does it make sense to return the existing project instance or do something else like throw an error or noop?